content/4_Gold/SP.problems.json

```diff
@@ -16,6 +16,18 @@
     }
   ],
   "dijk": [
+    {
+      "uniqueId": "cf-20C",
```

Comment:
I'm against adding this problem to this module/additional practice because it's a standard dijkstra problem.

Reply:
It repeats the idea of https://cses.fi/problemset/task/1671 (focus problem from the module).

Reply:
> It repeats the idea of https://cses.fi/problemset/task/1671 (focus problem from the module).

What about replacing https://cses.fi/problemset/task/1671 with this problem?

Reply:
why?

Reply:
why?

Reply:
> why?

They're basically the same problem, except https://cses.fi/problemset/task/1671 doesn't have finding the shortest path. I feel like finding the shortest path teaches you more idk.

Reply:
> why?

Did you repeat "why?" on purpose btw?

Reply:
oops bad internet

Reply:
> They're basically the same problem, except https://cses.fi/problemset/task/1671 doesn't have finding the shortest path. I feel like finding the shortest path teaches you more idk.

> idk, ask someone from core? 👀

Reply:
In general I like CSES better than CF as focus problem. But I didn't actually read the problems 😆